### PR TITLE
Use correct algorithm name for fixed limit backpressure 

### LINKED ISSUE #5257: Documentation shows different backpressure algorithm name
**Describe the bug**

Backpressure algorithm "FixedLimit" has an different enum then it is documented in the configuration.

It is documented in the configuration as:

```
  # The algorithm configures which algorithm to use for the backpressure.
      # It should be one of vegas, aimd, fixedLimit, gradient, or gradient2.
      # This setting can also be overridden using the environment variable ZEEBE_BROKER_BACKPRESSURE_ALGORITHM
      # algorithm: "vegas"
```

But if you use 

```
  - name: ZEEBE_BROKER_BACKPRESSURE_ALGORITHM
    value: "fixedLimit"
```

Then you get

```
{"severity":"INFO","logging.googleapis.com/sourceLocation":{"function":"logMessage","file":"ConditionEvaluationReportLoggingListener.java","line":136},"message":"\n\nError starting ApplicationContext. To display the conditions report re-run your application with 'debug' enabled.","serviceContext":{"service":"zeebe-broker","version":"zell-fixed-300-single-part-100-workflows"},"context":{"threadId":1,"threadPriority":5,"loggerName":"org.springframework.boot.autoconfigure.logging.ConditionEvaluationReportLoggingListener","threadName":"main"},"timestampSeconds":1598620848,"timestampNanos":765074000,"logger":"org.springframework.boot.autoconfigure.logging.ConditionEvaluationReportLoggingListener","thread":"main"}
{"severity":"ERROR","logging.googleapis.com/sourceLocation":{"function":"report","file":"LoggingFailureAnalysisReporter.java","line":40},"message":"\n\n***************************\nAPPLICATION FAILED TO START\n***************************\n\nDescription:\n\nFailed to bind properties under 'zeebe.broker.backpressure' to io.zeebe.broker.system.configuration.backpressure.BackpressureCfg:\n\n    Property: zeebe.broker.backpressure.algorithm\n    Value: fixedLimit\n    Origin: System Environment Property \"ZEEBE_BROKER_BACKPRESSURE_ALGORITHM\"\n    Reason: No enum constant io.zeebe.broker.system.configuration.backpressure.BackpressureCfg.LimitAlgorithm.FIXEDLIMIT\n\nAction:\n\nUpdate your application's configuration\n","serviceContext":{"service":"zeebe-broker","version":"zell-fixed-300-single-part-100-workflows"},"context":{"threadId":1,"threadPriority":5,"loggerName":"org.springframework.boot.diagnostics.LoggingFailureAnalysisReporter","threadName":"main","reportLocation":{"functionName":"report","filePath":"LoggingFailureAnalysisReporter.java","lineNumber":40}},"@type":"type.googleapis.com/google.devtools.clouderrorreporting.v1beta1.ReportedErrorEvent","timestampSeconds":1598620848,"timestampNanos":770094000,"logger":"org.springframework.boot.diagnostics.LoggingFailureAnalysisReporter","thread":"main"}
```

See here for example https://github.com/zeebe-io/zeebe/blob/develop/broker/src/main/java/io/zeebe/broker/transport/backpressure/PartitionAwareRequestLimiter.java#L60

**To Reproduce**

Use "fixedLimit" as algorithm.

**Expected behavior**
It works.

**Environment:**
 - OS: k8s
 - Zeebe Version: snapshot

### PR DESCRIPTION
**Pull request by menski**
## Description

Fixes the mismatch between documentation and implementation of the back pressure algorithm "fixed".

This is split out of the PR #5267

## Related issues

closes #5257 

## Definition of Done

_Not all items need to be done depending on the issue and the pull request._

Code changes:
* [ ] The changes are backwards compatibility with previous versions
* [ ] If it fixes a bug then PRs are created to [backport](https://github.com/zeebe-io/zeebe/compare/stable/0.24...develop?expand=1&template=backport_template.md&title=[Backport%200.24]) the fix to the last two minor versions

Testing:
* [x] There are unit/integration tests that verify all acceptance criterias of the issue
* [ ] New tests are written to ensure backwards compatibility with further versions
* [x] The behavior is tested manually
* [ ] The impact of the changes is verified by a benchmark 

Documentation: 
* [ ] The documentation is updated (e.g. BPMN reference, configuration, examples, get-started guides, etc.)
* [ ] New content is added to the [release announcement](https://drive.google.com/drive/u/0/folders/1DTIeswnEEq-NggJ25rm2BsDjcCQpDape)

### CODE REVIEW

File: dist/src/main/config/broker.standalone.yaml.template [hunk 1]
```diff
@@ -324,9 +324,9 @@
       # useWindowed: true
 
       # The algorithm configures which algorithm to use for the backpressure.
-      # It should be one of vegas, aimd, fixedLimit, gradient, or gradient2.
+      # It should be one of vegas, aimd, fixed, gradient, or gradient2.
       # This setting can also be overridden using the environment variable ZEEBE_BROKER_BACKPRESSURE_ALGORITHM
-      # algorithm: "vegas"
+      # algorithm: "fixed"
```
Comment (deepthidevaki) [inline dist/src/main/config/broker.standalone.yaml.template:329]:
```suggestion
      # algorithm: "vegas"
```

**Review by deepthidevaki — APPROVED** [review]
:+1: Please fix the default value in doc before merge.

File: dist/src/main/config/broker.standalone.yaml.template (hunk 1 above)
Comment (menski) [reply dist/src/main/config/broker.standalone.yaml.template:329]: I justed checked, and the default is still vegas in this PR https://github.com/zeebe-io/zeebe/pull/5317/files#diff-deea4dc9f416a8f4f5dd435a177d8b84R16. 

Is it okay if I merge @deepthidevaki ?

Comment (deepthidevaki) [reply dist/src/main/config/broker.standalone.yaml.template:329]: Yes. :+1:

**Pull request merged**